x-pack/plugins/screenshotting/jest.integration.config.js

```diff
@@ -5,4 +5,8 @@
  * 2.0.
  */
 
-export { PdfMaker } from './pdfmaker';
+module.exports = {
+  preset: '@kbn/test/jest_integration',
+  rootDir: '../../..',
+  roots: ['<rootDir>/x-pack/plugins/reporting'],
```

Comment:
I guess the path should be updated.
```suggestion
  roots: ['<rootDir>/x-pack/plugins/screenshotting'],
```